index.html

```diff
@@ -0,0 +1,116 @@
+        </section>
+        <section class="patterns">
+          <img src="assets/images/patterns/law-1.svg" class="pat-1" alt="some designs">
+          <img src="assets/images/patterns/law-1.svg" class="pat-2" alt="some designs">
+          <img src="assets/images/patterns/pat-1.jpg" class="pat-3" alt="some designs">
+          <img src="assets/images/patterns/pat-2.webp" class="pat-4" alt="some designs">
+          <img src="assets/images/patterns/pat-1.jpg" class="pat-5" alt="some designs">
+          <img src="assets/images/patterns/pat-2.webp" class="pat-6" alt="some designs">
+          <img src="assets/images/patterns/pat-1.jpg" class="pat-7" alt="some designs">
+          <img src="assets/images/patterns/currency-dollar.png" class="pat-8" alt="some designs">
+          <img src="assets/images/patterns/currency-dollar.png" class="pat-9" alt="some designs">
+          <img src="assets/images/patterns/redStripe.png" class="pat-10" alt="some designs">
+          <img src="assets/images/patterns/blueStripe.png" class="pat-11" alt="some designs">
+          <img src="assets/images/patterns/law-1.svg" class="pat-12" alt="some designs">
+        </section>
+    </section>
+    <section class="program">
+      <article class="mainP-heading">
+        <h2 class="main-text">Our Services</h2>
+        <img src="assets/images/patterns/underline.png" alt="underline for text">
+      </article>
+      <section class="service-holder">
+        <section class="service-1">
+          <img src="assets/images/patterns/employee-e.png" alt="service 1 image">
+          <h2 class="service-title">Consultation</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        <section class="service-2">
+          <img src="assets/images/patterns/blackboard-e.png" alt="service 1 image">
+          <h2 class="service-title">Internship</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        <section class="service-3">
+          <img src="assets/images/patterns/speech-bubble-e.png" alt="service 1 image">
+          <h2 class="service-title">arbitration</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        <section class="service-4">
+          <img src="assets/images/patterns/blackboard-e.png" alt="service 1 image">
+          <h2 class="service-title">Mediation</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        <section class="service-5">
+          <img src="assets/images/patterns/atom-e.png" alt="service 1 image">
+          <h2 class="service-title">Networking</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
```

Comment:
- [ ] please increase the heights of the cards a little so as to make them more visible and also so it looks like the original design. nice job so far.
your design
![Screenshot (1015)_LI](https://user-images.githubusercontent.com/65298048/234988026-24af0243-41ca-4f08-8227-11b9c209f2aa.jpg)
original
![Screenshot (1000)](https://user-images.githubusercontent.com/65298048/234988085-868603a5-8502-4bf7-be38-ebbf39b0ac5c.png)